Avalonia.Desktop.slnf

```diff
@@ -45,6 +45,7 @@
       "tests\\Avalonia.Base.UnitTests\\Avalonia.Base.UnitTests.csproj",
       "tests\\Avalonia.Benchmarks\\Avalonia.Benchmarks.csproj",
       "tests\\Avalonia.Controls.DataGrid.UnitTests\\Avalonia.Controls.DataGrid.UnitTests.csproj",
+      "tests\\Avalonia.Controls.ItemsRepeater.UnitTests\\Avalonia.Controls.ItemsRepeater.UnitTests.csproj",
```

Comment:
Unrelated change, but I noticed just now that I didn't include this project in the slnf when I separated out `ItemsRepeater` and thought it didn't warrant a separate PR.